org.kde.AwesomeBump.json

```diff
@@ -0,0 +1,46 @@
+{
+    "app-id": "org.kde.AwesomeBump",
```

Comment:
This ID suggests that the app is a KDE's project, which I guess is not the case.

Reply:
Hi, I need platform with Qt libraries and as far I know KDE platform has Qt and not Gnome, hence the name.  What do you recommend ?

Reply:
ID: com.github.kmkolasinski.AwesomeBump

Reply:
The app ID has nothing to do with what the app depends on, it's just an identifier.
If you have a website (or plan to create it), say, `awesomebump.org`, use it's reverse domain name in the app ID, e.g. `org.awesomebump.AwesomeBump`. Otherwise use github domain with your username, e.g. `com.github.kmkolasinski.AwesomeBump`.

Reply:
Thanks for the explanation, I will change the ID then.